docs/sql-ref-syntax-qry-select-pivot.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: global
+title: PIVOT Clause
+displayTitle: PIVOT Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`PIVOT` clause is used for data perspective, we can get the aggregated values based on specific column value.
+
+### Syntax
+
+```sql
+PIVOT ( { aggregate_Expression [ AS aggregate_Expression_Alias ] } [ , ... ] FOR column_List IN ( expression_List ) )
```

Comment:
Could you follow the naming formats with the other pages: e.g., `aggregate_Expression ` -> `group_expression`?